### PR TITLE
migrate `TranslationBannerLegal` to Chakra

### PR DESCRIPTION
**Pull request by amit-ksh**
## Description

Migrate the `TranslationBannerLegal` component from emotion to Chakra.

### Original Style vs Current Style
hover Style

## Related Issue

#8632

### CODE REVIEW

File: src/components/TranslationBannerLegal.tsx [hunk 1]
```diff
@@ -125,20 +42,74 @@ const TranslationBannerLegal: React.FC<IProps> = ({
...
+              bgColor="primary"
+              color="buttonColor"
+              border="1px solid transparent"
+              _hover={{
+                bgColor: "primaryHover",
+                boxShadow: cardBoxShadow,
+              }}
+              _active={{
+                bgColor: "primaryActive",
+              }}
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:112]: You can remove these styles. The default Button is going to render the `solid` variant with the correct color scheme. You can see the details [here](https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/components/Button.ts).

```suggestion
            <Button
```

File: src/components/TranslationBannerLegal.tsx [hunk 2]
```diff
@@ -1,102 +1,18 @@
...
-`
+import {
+  Box,
+  Button,
+  CloseButton,
+  Flex,
+  Heading,
+  Text,
+  useToken,
+} from "@chakra-ui/react"
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:15]: import order. Can you put this import on the top of the file? below the react import

File: src/components/TranslationBannerLegal.tsx [hunk 3]
```diff
@@ -125,20 +42,74 @@ const TranslationBannerLegal: React.FC<IProps> = ({
...
+      zIndex="99"
+    >
+      <Flex
+        bg="infoBanner"
+        color="black300"
+        justify="space-between"
+        maxW={{ base: "full", md: "600px" }}
+        maxH="full"
+        p={4}
+        borderRadius="2px"
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:59]:
```suggestion
        borderRadius="sm"
```

File: src/components/TranslationBannerLegal.tsx [hunk 4]
```diff
@@ -125,20 +42,74 @@ const TranslationBannerLegal: React.FC<IProps> = ({
...
+        boxShadow={{
+          base: `-4px 10px 0px ${text} 10%`,
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:61]: nice 👍🏼

**Review by pettinarip — CHANGES_REQUESTED** [review]
@amit-ksh thanks man, this is looking great. I left a couple of small change requests.

File: src/components/TranslationBannerLegal.tsx (hunk 4 above)
Comment (amit-ksh) [reply src/components/TranslationBannerLegal.tsx:61]: Thanks!

**Pull request merged**